screens/LoginScreen.tsx

```diff
@@ -1 +1,49 @@
-// login screen
+import { ParamListBase } from "@react-navigation/native";
+import { NativeStackScreenProps } from "@react-navigation/native-stack";
+import { Button, StyleSheet, Text, View } from "react-native";
+
+import { colors } from "../constants/Colors";
+import { screen_names } from "../constants/ScreenNames";
+import { useAppDispatch, useAppSelector } from "../hooks/useTypedRedux";
+import { setUser } from "../state/user";
+
+type LoginScreenProps = NativeStackScreenProps<
+  ParamListBase,
+  screen_names.LOGIN,
+  undefined
+>;
```

Comment:
I am not sure about defining screen prop types on a seperate file as this type is only used in this screen.So  I think this can stay as it is. 
What do you think about this @makunomark

Reply:
Imo, there's no need to extract it to a separate file as its only used here.